README.md

```
@@ -48,6 +49,32 @@ pnpm install
 pnpm tauri build
 ```
 
+### Nix for macOS
+
+A `flake.nix` is provided for building and installing TaurusViewer on macOS (Apple Silicon / `aarch64-darwin`) via Nix:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)